Gopkg.toml

```diff
@@ -0,0 +1,70 @@
+
+# Gopkg.toml example
```

Comment:
Is this intentionally here ? ( the example part)

Reply:
Yep, it is auto-generated as part of the `dep` bootstrapping process. I think for now at least, because it is a new format for most folks that it makes sense to keep it here.

Reply:
Synced with Martin offline we can leave this part for a while as it is a new format